src/message.ts

```diff
@@ -604,6 +604,34 @@ export class Message implements Sayable {
     ])
   }
 
+  /**
+   * Return the recalled message id
+   *
+   * Valid only if {@link Message#type} is equal to `MsgType.RECALLED`
+   *
+   * @returns {string} msgId
+   * @memberof Message
+   * @example
+   * ```javascript
+   * Bot.on('message', async m => {
+   *   if (m.type() === MsgType.RECALLED) {
+   *     await m.say(`You recalled a message, message ID：${m.getRevokeId()}`)
+   *   }
+   * })
+   * ```
+   */
+  public getRevokeId(): string {
```

Comment:
RECALLED msg, content:
```json
&lt;sysmsg type="revokemsg"&gt;&lt;revokemsg&gt;&lt;session&gt;binsee&lt;/session&gt;&lt;oldmsgid&gt;1056815127&lt;/oldmsgid&gt;&lt;msgid&gt;6991717488370193149&lt;/msgid&gt;&lt;replacemsg&gt;&lt;![CDATA["杉木" 撤回了一条消息]]&gt;&lt;/replacemsg&gt;&lt;/revokemsg&gt;&lt;/sysmsg&gt;
```

use `UtilLib.unescapeHtml(this.obj.content)`
```xml
// the content has been formatted
<sysmsg type="revokemsg">
    <revokemsg>
        <session>binsee</session>
        <oldmsgid>1056815127</oldmsgid>
        <msgid>6991717488370193149</msgid>
        <replacemsg>
            <![CDATA["杉木" 撤回了一条消息]]>
        </replacemsg>
    </revokemsg>
</sysmsg>
```

We need to use getRevokeId () package to extract the code `msgId`

```javascript
const ret = UtilLib.unescapeHtml(this.obj.content).match(/<msgid>(\w+)<\/msgid>/)
```